src/recipes/RecipeEditBase.js

```diff
@@ -0,0 +1,91 @@
+import {v4 as uuid} from "uuid"
+import {useState} from "react";
+import useInputState from "./useInputState";
+import IngredientEdit from "./IngredientEdit";
+import {useNavigate} from "react-router-dom";
+
+export default function RecipeEditBase(props) {
```

Comment:
:thinking: I wouldn't split this in 2 files, I don't think we need the extra abstraction. 
Also `RecipeEditBase` doesn't really explain what it does to me, it rather hints at being a base class of some sort.
I think the common convention is to name the other component - if you need it - with the `Wrapper` suffix, as in `EditRecipeWrapper`.

Reply:
Right, `EditRecipeWrapper` sounds better actually. I think I took the `Base` word from Python. Actually, there are 2 screens which use `RecipeEditBase`, one for creation and another for editing, but yeah I think there can be some improvements with naming and code reuse.